config/initializers/two_factor_auth_controller_override.rb

```diff
@@ -18,6 +18,10 @@ def handle_two_factor_authentication
         end
       end
 
+      def teaspoon_controller?
```

Comment:
if it were me I would consider monkeypatching the `Teaspoon::SuiteController` in test (that way we keep test configs/settings from leaking into production?

ex in a `spec/support/` file:

```
class Teaspoon::SuiteController
  skip_before_filter :handle_two_factor_authentication
end
```